app/build.gradle

```diff
@@ -33,7 +33,7 @@ dependencies {
     api "info.picocli:picocli:${picoCliVersion}"
 
     testImplementation "org.junit.jupiter:junit-jupiter:${junitJupiterVersion}"
-    testImplementation "com.github.stefanbirkner:system-lambda:1.2.1"
+    testImplementation "uk.org.webcompere:system-stubs-core:2.0.2"
```

Comment:
We need to be a bit careful with new dependencies.

I don't know if we can take this one as it is dependent on https://mvnrepository.com/artifact/com.github.spotbugs/spotbugs-annotations which is LGPL licensed, which I think can be problematic.

@nargas-ritu may be able to confirm.

Meanwhile, do we need this, or is there a way we can work around it without bringing in a new dependency?

Reply:
Agreed! LGPL is a problematic one. If we can have a workaround, that will be better.

Reply:
The reason why I used this dependency is as follows:
> I found an alternative to system lambda that is not compatible with JVM 16 onward. [Here is the link](https://github.com/webcompere/system-stubs). This is basically a fork of System Lambda. This appears to be working without having to change the existing testing logic.

This dependency is used to set environment variables in tests.
I have implemented one way of doing it, using `System.setProperty()`:
https://github.com/corda/corda-cli-plugin-host/pull/184/commits/48be2ebed6eab15b080b08dec5a9ca650a4172c9
But one test (`FilesTest.testCliHomeDirWithEnvVar`) is explicitly testing an environment variable, not the property. I'm not sure if this test is meaningful in this case